terraform/050-pw-manager/vars.tf

```diff
@@ -75,14 +75,17 @@ variable "mailer_usefiles" {
 
 variable "mailer_host" {
   type = "string"
+  default = ""
```

Comment:
Do you think these empty-string defaults are going to cause us a problem? For all I remember I added these, forgetting about our env-var-without-a-value problem re: cron files.

Reply:
hmm, good question, it might. I'd rather put dummy values in the environment than the TF for now though until we solve the env issue in cron